src/photos/ducks/upload/index.jsx

```diff
@@ -70,11 +75,24 @@ const processNextFile = callback => async (dispatch, getState) => {
     dispatch({ type: RECEIVE_UPLOAD_SUCCESS, file })
   } catch (error) {
     logger.log(error)
+    const statusError = {
+      409: CONFLICT,
```

Comment:
It doesn't look like CONFLICT is being used here at all?

Reply:
thanks !